### DISCUSSION TITLE
Progetto open source Luigi Gorgone

### DISCUSSION THREAD

**Discussion by luishasleft** [deneb-alpha/Lab_open_source-2023_2024#26] [Projects]
**Parte principale:**
-Modelli di business nel mondo open source
-Creazione di una coppia di chiavi SSH ed integrazione in github
-Risoluzione 4 bug con bugzilla
-->riproduzione di un bug

**Comment by deneb-alpha** [comment]
Ciao @luishasleft,
come idea generale ci siamo, ti chiederei se possibile di presentare anche la tua esperienza d'interazione con il gruppo QA di LibreOffice e come hai trovato i materiali guide/wiki disponibili ai nuovi contributori. Come sempre, anche critiche negative ;)
